app/renderer/js/pages/preference/general-section.ts

```diff
@@ -130,11 +138,20 @@ export default class GeneralSection extends BaseSection {
 				<div class="title">${t.__('Reset Application Data')}</div>
                 <div class="settings-card">
 					<div class="setting-row" id="resetdata-option">
-						<div class="setting-description">${t.__('This will delete all application data including all added accounts and preferences')}
+						<div class="setting-description">${t.__('This will reset all application settings to default preserving added organisations and accounts')}
 						</div>
 						<button class="reset-data-button red w-150">${t.__('Reset App Data')}</button>
 					</div>
 				</div>
+				<div class="title">${t.__('Factory Reset')}</div>
+                <div class="settings-card">
+					<div class="setting-row" id="factory-reset-option">
+						<div class="setting-description">${t.__('This will delete all connected organisations, accounts, certificates and preferences')}
```

Comment:
"Reset all desktop application settings to defaults, deleting connected organizations and accounts"